action.yml

```diff
@@ -14,6 +14,10 @@ inputs:
   upload-chunk-size:
     description: 'The chunk size used to split up large files during upload, in bytes'
     required: false
+  skip-save:
+    description: 'Whether or not to skip the saving of the cache.  If this is set to "true", the cache will not be saved.  It can also be set to an evironment variable such as "\$\{\{ env.MY_SKIP_SAVE \}\}".  If the MY_SKIP_SAVE environment variable is "true" by the end of the CI, caching will be skipped.  Default is to update if and only if there was no primary-key exact cache hit.'
```

Comment:
Since this comment is really long, can you use `>` to split it up so it's more readable.

Here is an example: https://github.com/actions/checkout/blob/25a956c84d5dd820d28caab9f86b8d183aeeff3d/action.yml#L12-L24